### PR TITLE
Remove SweepAround.Us references

### PR DESCRIPTION
**Pull request by srobbin**
- Change logo
- Move site name to an ENV
- Removed other references to sweeparound.us

### CODE REVIEW

**Comment by srobbin** [comment]
@KatherineEvans, @carlhauck  Wasn't sure if it was going to be "WeTheSweeple" or "We The Sweeple". If I got it wrong, let me know and I can change the logo.

File: .env.example [hunk 1]
```diff
@@ -6,6 +6,7 @@ MAILGUN_API_KEY=""
 MAILGUN_DOMAIN=""
 SECRET_KEY_BASE=""
 SECRET_KEY_JWT=""
+SITE_NAME=SweepAround.Us
```
Comment (carlhauck) [inline .env.example:9]: SITE_NAME="We The Sweeple"

And for the logo, I think it'd make sense to jam the words up against each other like you had for the SweepAround.Us logo.

**Comment by carlhauck** [comment]
@srobbin I left just one comment about two small things. Everything else looks great, though!

File: .env.example (hunk 1 above)
Comment (KatherineEvans) [reply .env.example:9]: Agreed!

Comment (srobbin) [reply .env.example]: Done! Just pushed a commit.

**Pull request merged**